src/js/player.js

```diff
@@ -300,7 +300,17 @@ class Player extends Component {
     if (tag.parentNode) {
       tag.parentNode.insertBefore(el, tag);
     }
-    Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    //Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    // append the DOM el function to make use of our component.addChild method
+    // to properly be injected into the children Array
+    tag.el = function () {
```

Comment:
Looks like the issue is that for some reason if we're using `Dom.insertElFirst` here, the video element ends up becoming the last child of the player div. If we're doing this `tag.el()` thing, it ends up working correctly.
Investigating.